src/lib/accessibility/Behaviors/Menu/MenuItemBehavior.ts

```diff
@@ -11,6 +12,14 @@ const MenuItemBehavior: Accessibility = (props: any) => ({
       tabIndex: '0',
     },
   },
+
+  actionsDefinition: {
```

Comment:
maybe we could use simpler name like just `actions`?

Reply:
Yeah, but it can break the meaning of it, especially during mapping real actions of a component to these. As for me, this name really means what it does.